Lib/fontTools/colorLib/builder.py

```diff
@@ -377,7 +378,9 @@ def _reuse_ranges(num_layers: int) -> Generator[Tuple[int, int], None, None]:
         # Reuse of very large #s of layers is relatively unlikely
```

Comment:
the TODO above no longer applies after you did add a max length to limit the scope of search.
Curious how did you come up with 32? I wonder if we could/should make this a build-time option that user could tweak?